tests/e2e/README.md

```diff
@@ -12,13 +212,26 @@ When adding a new proposal type:
 * add a case for your proposal type to `json_utils.go/UnmarshalProposalWithType`
 * add a generator for your proposal type in `state_rapid_test.go` and add it to the `GetProposalGen` function
 
-## Regenerating Traces
+### Regenerating Traces
 
 The traces in `tracehandler_testdata` are generated by the test `trace_handlers_test.go/TestWriteExamples`.
 
 You can regenerate them by running `make e2e-traces` in the root of the repo.
 
-## Running against traces
+### Running against traces
 
 To run a test trace in the e2e tests, run `go run . --test-file $TRACEFILE::$TESTCONFIG`.
 See `--help` for more details.
+
+## Debugging and using the e2e infrastructure to set up a testnet
+
+When something in the tests goes wrong, a nice thing about the tests is that the
+docker container doesn't get killed.
```

Comment:
Just to verify. My understanding was that we needed to comment out `tr.teardownDocker()` in [main.go](https://github.com/cosmos/interchain-security/blob/ph/e2e-tutorial/tests/e2e/main.go#L319) so that the Docker containers do not get killed. Is that not the case?

Reply:
They remain present *if* a test fails. If you want them to remain always present, commenting teardownDocker does it